### DISCUSSION TITLE
500 http error by code in connection.php

### DISCUSSION THREAD

**Discussion by Alinoor9999** [laravel/framework#45284] [Q&A]
throw new QueryException($query, $this->prepareBindings($bindings), $e
hi community, I found the above code in my connection.php file script that is not letting the website to load as https and it gives 500 error

**Comment by henzeb** [comment]
Then I suggest you look at the failed query, find out what value is missing, and where it comes from.

you can also share the error message with us, so we can try helping you find out what might go wrong.
